corehq/apps/reports/standard/cases/basic.py

```diff
@@ -24,6 +26,129 @@
 from .data_sources import CaseInfo, CaseDisplay
 
 
+class CaseExportScope(object):
```

Comment:
Right.. that sounds so right.. 
The changes are mostly for restricted user access like not allowing filtering by user types, groups etc or adding a scope filter for restricted user. For a user that can access all locations, i think all remains the same plus the locations integration is added.